backend/app.js

```diff
@@ -1,21 +1,34 @@
 const express = require("express");
 const path = require("path");
+const mongoose = require("mongoose");
 const app = express();
 
 const indexRouter = require("./routes/indexRouter");
 const chatRouter = require("./routes/chatRouter");
 const authRouter = require("./routes/authRouter");
 
-// Serve static files from the frontend folder globally
 app.use(express.static(path.join(__dirname, "../frontend/public")));
 
+const mongoURI = "mongodb://localhost:27017/myDatabase";
+
+mongoose.connect(mongoURI, { useNewUrlParser: true, useUnifiedTopology: true })
+  .then(() => {
+    console.log("Connected to MongoDB on localhost");
+  })
+  .catch((err) => {
+    console.log("Error connecting to MongoDB:", err);
+  });
+
 app.use("/auth", authRouter);
 app.use("/chat", chatRouter);
 app.use("/", indexRouter);
 
-// Error handling for unmatched routes
 chatRouter.get("*", (req, res) => {
-    res.sendFile(path.join(__dirname, "../../frontend/public/error.html"));
+  res.sendFile(path.join(__dirname, "../../frontend/public/error.html"));
 });
 
 module.exports = app;
+
+app.listen(3000, '127.0.0.1', () => {
+    console.log('Server is running at http://127.0.0.1:5000/');
+});
```

Comment:
I believe that we already have these lines in server.js

Reply:
also there's no server running at port 5000 since app only listens at port 3000 now